src/components/AxisCollection/CollapsedAxis.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react';
+import * as d3 from 'd3';
+import PropTypes from 'prop-types';
+import isEqual from 'lodash.isequal';
+import { createYScale } from '../../utils/scale-helpers';
+import { singleSeriePropType } from '../../utils/proptypes';
+
+export default class CollapsedAxis extends Component {
+  static propTypes = {
+    // TODO: Zooming is currently not supported.
+    // It might be nice to support zooming all of the axes when you zoom on
+    // the collapsed one. Experiment with this and either add it or remove
+    // all of this unused code.
+    zoomable: PropTypes.bool,
```

Comment:
Completely agree, having an all-series zoom feature through a collapsed axis would be really nice. It can be moved to another issue though.

Reply:
Created a new issue for this #66.